.github/workflows/acceptance.yml

```diff
@@ -324,8 +324,8 @@ jobs:
       - name: Diff snapshots
         id: visual-snapshots-diff
         uses: getsentry/action-visual-snapshot@v2
-        # Do not execute on forks or on master. Forks are handled in visual-diff.yml
-        if: github.head_repository.full_name == 'getsentry/sentry' && github.ref != 'refs/heads/master'
+        # Do not execute on forks. Forks are handled in visual-diff.yml
+        if: github.event.pull_request.head.repo.full_name == 'getsentry/sentry' || github.ref == 'refs/heads/master'
```

Comment:
I've pushed one more commit.
This needs to run on PRs within Sentry or a master push.
`visual-diff` is exclusive for PRs from forks.

This [push](https://github.com/getsentry/github-actions-testing/actions/runs/2227071984) on `master` shows this running and [workflow_run not executing](https://github.com/getsentry/github-actions-testing/actions/runs/2227075158):
<img width="166" alt="image" src="https://user-images.githubusercontent.com/44410/165316384-733149aa-c046-40cd-b013-c4089b8b2e66.png">
<br />
<img width="146" alt="image" src="https://user-images.githubusercontent.com/44410/165316432-c2433a58-1bcb-4ee5-a0aa-7fbd24e0740d.png">